include/sound_tool.h

```diff
@@ -0,0 +1,105 @@
+#ifndef POKEHEARTGOLD_SND_TOOL_H
+#define POKEHEARTGOLD_SND_TOOL_H
+
+#include <nnsys/snd/waveout.h>
+#include <nnsys/snd/capture.h>
```

Comment:
These should instead be included in <nnsys/snd.h>. No need to explicitly include the nnsys headers here, as they are included in "global.h" which is passed via commandline.